chatterbot/corpus/data/english/history.corpus.json

```diff
@@ -0,0 +1,37 @@
+{
+
+    "history": [
+        [
+            "american civil war",
```

Comment:
"american civil war" looks like a topic, when the text here should probably be a question or something else.